src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -388,6 +412,44 @@ private Expression CreatePropertyAccessExpression(Expression source, IEdmPropert
             }
         }
 
+        private Expression CreateOpenPropertyAccessExpression(SingleValueOpenPropertyAccessNode openNode)
+        {
+            var sourceAccessor = BindAccessor(openNode.Source);
```

Comment:
>var s [](start = 12, length = 5)

var hides the actual type when using tools like code flow or notepad.

Reply:
Are we going to get rid of vars in tests too?